umm-lib/test/cmr/umm/test/echo10/granule.clj

```diff
@@ -24,8 +24,15 @@
   (-> gran
       ;; Update the related-urls as ECHO10 OnlineResources' title is built as description plus resource-type
       (update-in [:related-urls] tc/umm-related-urls->expected-related-urls)
+      ;; Remove crid-ids and feature-ids from umm-lib granule since they are not supported in echo10.
+      (update-in [:data-granule] dissoc :crid-ids :feature-ids)
```

Comment:
```suggestion
      (update-in [:data-granule] assoc :crid-ids nil :feature-ids nil)
```

Reply:
Changed to: (update-in [:data-granule] #(if % (assoc % :crid-ids nil :feature-ids nil) %))